config.yml

```diff
@@ -1,584 +1,364 @@
+    event: pull_request.synchronize
+    link: '{{ repoUrl }}/pull/1'
+    actions: 
+    # Step 3: Merge the pull request
+    - type: removeBranchProtection
     - type: respond
-      with: 12.1_e-no-meta.md
-  - type: requestReviewFromRegistrant
-  - type: respond
-    with: approve-first-pr.md
+      with: 03_merge.md
+      data:
+        url: '%payload.repository.html_url%/edit/course-planning/config.yml?'
 
-# 4
-- title: Approve the course metadata
-  description: Approve the pull request containing the course's metadata.
-  link: '{{ repoUrl }}/pull/1'
-  event: pull_request_review.submitted
-  actions:
-  - type: gate
-    left: '%payload.review.state%'
-    operator: ===
-    right: approved
-  - type: mergePullRequest
-  - type: createPullRequest
-    title: Setup the learner's repo
-    body: 02.0_add-template.md
-    head: repo-setup
-    action_id: repoPR
-  - type: respond
-    with: goto-template-pr.md
-    data:
-      url: '%actions.repoPR.data.html_url%'
-  - type: assignRegistrant
-    issue: '%actions.repoPR.data.number%'
-  - type: octokit
-    method: pullRequests.createComment
-    owner: '%actions.repoPR.data.head.repo.owner.login%'
-    repo: '%actions.repoPR.data.head.repo.name%'
-    number: '%actions.repoPR.data.number%'
-    commit_id: '%actions.repoPR.data.head.sha%'
-    path: config.yml
-    body: |
-      ```suggestion
-        repo: lab-starter-template
-      ```
-    position: 7
+  - title: Merge your learning objectives
+    description: Merge the pull request breaking down your course topic into smaller learning objectives.
+    event: pull_request.closed
+    link: '{{ repoUrl }}/pull/1'
+    actions:
+    - type: gate
+      every: true
+      gates:
+      - left: '%payload.action%'
+        operator: ===
+        right: closed
+      - left: '%payload.pull_request.merged%'
+    - type: respond
+      with: 03_nice-work.md
+    # Step 4: Choose a template project
+    - type: createPullRequest
+      title: Choose a template project
+      body: 04_choose-a-template.md
+      head: template
+      data:
+        url: '%payload.repository.html_url%/edit/template/config.yml?'
+    - type: updateBranchProtection
 
-# 5
-- title: Point to the template repo
-  description: Designate a template repository from which to clone.
-  link: '{{ repoUrl }}/pull/2'
-  event: pull_request.synchronize
-  actions:
-  - type: getFileContents
-    action_id: fileContents
-    filename: 'config.yml'
-  - type: gate
-    left: '/^\s{2}repo: *lab-starter-template(?!#).*$/gm'
-    operator: test
-    right: '%actions.fileContents%'
-    else:
+  - title: Reference the template
+    description: Edit the config file to reference your chosen template repository.
+    event: pull_request.synchronize
+    link: '{{ repoUrl }}/pull/2'
+    actions:
+  # - type: gate # we validate that the repository exists
+    - type: gate
+      left: "%actions.pr.data.mergeable%"
+      else:
```

Comment:
@brianamarie we might need to check this gate, I received a message that I had a merge conflict even though I didn't not have a conflict. Does the `mergeable` also return false if there is a review required?

<img width="762" alt="Screen Shot 2020-01-10 at 10 26 43 AM" src="https://user-images.githubusercontent.com/9950121/72164475-bf3ee280-3393-11ea-8013-854b201be4ae.png">